packages/common/src/types/UserCredential.ts

```diff
@@ -0,0 +1,4 @@
+export type UserCredential = {
```

Comment:
Move this file to a new folder called `requests` and call it the request method name + "Request"

```suggestion
export type SignInRequest = {
```